bugbot/rules/triage_owner_rotations.py

```diff
@@ -87,17 +104,65 @@ def get_email_data(self, date: str) -> List[dict]:
         failures = self._update_triage_owners(new_triagers)
         self.has_put_errors = len(failures) > 0
 
-        return [
-            {
-                "component": new_triager.component,
-                "old_triage_owner": self.component_triagers.get_current_triage_owner(
-                    new_triager.component
-                ),
-                "new_triage_owner": new_triager.bugzilla_email,
-                "has_put_error": new_triager.component in failures,
-            }
-            for new_triager in new_triagers
-        ]
+        email_data = []
+
+        for new_triager in new_triagers:
+            old_owner = self.component_triagers.get_current_triage_owner(
+                new_triager.component
+            )
+
+            email_data.append(
+                {
+                    "component": new_triager.component,
+                    "old_triage_owner": old_owner,
+                    "new_triage_owner": new_triager.bugzilla_email,
+                    "has_put_error": new_triager.component in failures,
+                }
+            )
+
+        return email_data
```

Comment:
Why do we need to refactor this?

Reply:
Forgot to revert these changes, removed the refactoring just now!